kensu/psycopg2/extras/__init__.py

```diff
@@ -0,0 +1,51 @@
+from kensu.psycopg2.parser import parse_and_report
+from psycopg2.extras import *
+import psycopg2
+
+# FIXME: now it's copy paste now, but we kinda need that...
```

Comment:
@vidma not sure you need this copy paste, you could call the original method from within this extension, as it is done for pandas read_* right?

I get this is because of the diff form `sql` can take, although I would "only" extract the beginning that handles this thing and not the rest?
If this is not as easy, let's move on with this first or only accept string for now :-D

cc @SammyEK

Reply:
this is a validating parser, so safest and simplest is to build real SQL for first page of data.
the actual DB querying and result obtaining is indeed fully delegated already (only we convert argslist to a `list`, in case it was iterable-once)